build.gradle.kts

```diff
@@ -4,21 +4,20 @@ buildscript {
     repositories {
         google()
         mavenCentral()
-        jcenter()
+        gradlePluginPortal()
```

Comment:
I'd prefer the order:
1. maven central
2. gradle plugin portal (do we need this one?)
3. google

Reply:
Apparently it's unneeded, it was for an older version of the JUnit plugin after removing JCenter..